src/main/java/com/keeper/homepage/domain/attendance/dto/response/AttendanceRankResponse.java

```diff
@@ -0,0 +1,36 @@
+package com.keeper.homepage.domain.attendance.dto.response;
+
+import static lombok.AccessLevel.PRIVATE;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.keeper.homepage.domain.attendance.entity.Attendance;
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+@AllArgsConstructor(access = PRIVATE)
+public class AttendanceRankResponse {
+
+  private Integer rank;
+  private String thumbnailPath;
+  private String nickName;
+  private Float generation;
+  private Integer continuousDay;
+
+  @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+  private LocalDateTime time;
+
+  public static AttendanceRankResponse from(Attendance attendance) {
+    return AttendanceRankResponse.builder()
+        .rank(attendance.getRank())
+        .thumbnailPath(attendance.getMember().getThumbnailPath())
+        .nickName(attendance.getMember().getNickname())
+        .generation(attendance.getMember().getGeneration())
```

Comment:
```suggestion
        .generation(Float.toString(attendance.getMember().getGeneration()))
```

실수형을 표기하는 방법이 다 달라서 서버에서 String으로 넘겨주는게 좋을 것 같더라구요~! (generation을 실수형으로 만든게 큰 패착입니다... 혹시 남는 인원 있으면 generation부터 string으로 바꿔주세용... ㅠㅠ)

Reply:
헐 generation 닉네임과 함께 String으로 바꾸겠습니다!!

Reply:
@gusah009 으흠.. 생각해보니 컬럼의 타입을 변경하기 보다 `getGeneration` 메서드가 `String`을 반환하도록 변경하는 건 어떨까요?
`Generation` 객체 자체에 이미 로직도 구현이 되어있고..ㅎㅎ